### ISSUE TITLE
Profile that is disliked on a matching stack is being showed on other events matching stack

### ISSUE THREAD

**Issue by RajeevReddyIlavala** [gcivil-nyu-org/fall2019-cs-gy-6063-team-debug_entities#233]
## Current Functionality
Even if I dislike a particular user on an event matching stack, the same user is being displayed on other events matching stack

## Expected Functionality
If you dislike an user in one matching stack, the same user shouldn't be displayed on other events matching stack

## Notes
Not sure if you implemented the current functionality intentionally, if so then at least there should be a feature to avoid permanently some users or else after using the app for a while the user will end up having lengthy matching stacks with profiles that the user don't even want to take a look.

**Comment by mksavic** [comment]
Yes, this was implemented purposefully. But, I agree that the user should have the option to never see a squad ever again, even in other matching stacks. Moving to Icebox.
